scripts/constants.py

```diff
@@ -51,8 +57,10 @@ class ChainId(Enum):
 
 
 CHAIN_ID = getattr(ChainId, NETWORK)
+KAKAROT_CHAIN_ID = 1263227476  # KKRT (0x4b4b5254) in ASCII
 
-DEPLOYMENTS_DIR = Path("deployments") / NETWORK
+DEPLOYMENTS_DIR = Path("deployments")
+DEPLOYMENTS_NETWORK_DIR = Path("deployments") / NETWORK
```

Comment:
I would have kept the `DEPLOYMENTS_DIR` ("deployments" / NETWORK) as the main exposed constants as you don't really want to think about network elsewhere than when you set the STARKNET_NETWORK env variable.